src/pages/my-decade-in-review/index.es.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Mi década en retrospectiva
+date: '2020-01-01'
+spoiler: Una reflexión personal.
+---
+
+Empecé esta década como un estudiante de primer año salido de la prepa. Tenía 17, no tenía un trabajo, no tenía conexiones en la industria, y no sabía un carajo. Y ahora estás leyendo mi blog! Hubiera estado orgulloso.
+
+He contado pedazos de mi historia en diferentes podcasts. Ahora siento que es el momento adecuado de escribir las partes más memorables para mi.
+
+La historia de cada persona es única y no es directamente reproducible. Me beneficié inmensamente por el privilegio de haber nacido en una familia de clase media y el aspecto de un típico programador. Las personas se arriesgaban conmigo. Aún así, espero que compartiendo mi historia sea útil para comparar nuestras experiencias. Incluso si nuestras circunstancias fueron muy diferentes, al menos quizás encuentres parte de ellas entretenidas.
+
+
+## 2010
+
+Nací en Rusia y teminé la prepa allá en 2009. En Rusia, la educación superior es gratis si sacas buenas notas en los exámenes. Probé con varias escuelas. Quería particularmente entrar a un colegia donde sus estudiantes habitualmente ganan concursos de programación (que creía que genial en ese momento).
```

Comment:
`Probe con varias instituciones`, from the common view of the word `escuela` it can understand it as not a University.

`colegia` -> `colegio`.
And if accept the first suggestion of changing the word, would be `colegia` change to `intitucion`.

`(que creía que genial en ese momento).` -> `(que creía genial en ese momento).`